### DISCUSSION TITLE
Judecoin China Community

### DISCUSSION THREAD

**Discussion by laogongtou** [Judecoin/judecoin#10] [General]
Hello, I am the co-founder of Judecoin China community. We are very optimistic about Judecoin's atomic exchange technology and believe that this technology will have a great future in the blockchain industry. Therefore, there are many holders in our Chinese community. Due to the lack of latest news in our community, we are currently actively seeking Judecoin development teams. We hope that the development team can provide us with some information and technical support so that our community can develop even bigger and better. Judecoin, as a leader in future land application in the web era, is also an inseparable part of the community. We can actively promote and build more consensus, and hope to contact the development team, thank you。
My email：18746062634@163.com

**Comment by chrisrobert555** [comment]
Hello, thank you for reaching out. We appreciate your enthusiasm and [support](https://www.cottonwooddentalgroup.com/) for Judecoin's technology. We will connect you with our development team to provide the information and technical support your community needs.

**Reply by ghost** [reply]
Hello, may I ask how to communicate with your Judecoin development team? We will continue to communicate with you in our Chinese community. Thank you
